modules/installation/proc-installing-rhdh-on-kubernetes-with-helm-cli.adoc

```diff
@@ -0,0 +1,58 @@
+[id="installing-rhdh-on-kubernetes-with-helm-cli_{context}"]
+= Installing {product} on Kubernetes with Helm CLI
+
+You can use the Helm CLI to install {product} on Kubernetes.
+
+.Prerequisites
+* A `kubectl` session, with `developer` or `admin` permissions, on a Kubernetes cluster.
+* You have installed the Helm CLI.
+
+.Procedure
+. Add the OpenShift Helm Charts repository.
++
+----
+helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+
+. Create and activate the _<rhdh>_ Kubernetes namespace:
++
+[subs="quotes+"]
+----
+NAMESPACE=_<rhdh>_
+kubectl create namespace ${NAMESPACE} || true
+kubectl config set-context --current --namespace=${NAMESPACE}
+kubectl config view --minify | grep namespace:
+----
+
+. Install the {product-short} Helm Chart:
++
+[subs="attributes+"]
+----
+helm upgrade redhat-developer-hub -i https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz
+----
+
+. Collect values from your OpenShift or Kubernetes cluster:
++
+----
+PASSWORD=$(kubectl get secret redhat-developer-hub-postgresql -o jsonpath="{.data.password}" | base64 -d)
+CLUSTER_ROUTER_BASE=$(kubectl get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
```

Comment:
@nickboldt I need a kubectl equivalent here

Reply:
does the above not work? it worked for me using kubectl version:
```
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.0", GitCommit:"45460a55f50834ff18308b13b4569a6c89b9d37a", GitTreeState:"clean", BuildDate:"2022-06-27T10:14:57Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
```